backend/lib/services/infrastructureSecrets.js

```diff
@@ -196,12 +189,16 @@ exports.list = async function ({ user, namespace }) {
 }
 
 exports.create = async function ({ user, namespace, body }) {
-  const secret = await Core(user).namespaces(namespace).secrets.post({
-    body: toSecretResource(body)
+  const client = user.api
+
+  const secret = await client.core.secrets.create({
```

Comment:
add assertions for all http resource methods which ensure required parameters are specified correctly to prevent path traversal attacks

Reply:
✔️